### ISSUE TITLE
ResourceWarning: unclosed file VERSION

### ISSUE THREAD

**Issue by grigi** [CleanCut/green#200]
When running with `PYTHONWARNINGS=default` envvar, I get the following:
```py
.pyenv/lib64/python3.6/site-packages/green/version.py:7: ResourceWarning: unclosed file <_io.TextIOWrapper name='.pyenv/lib64/python3.6/site-packages/green/VERSION' mode='r' encoding='UTF-8'>
```
It seems that green isn't closing its own VERSION file.

Changing line 7 to:
```py
with open(os.path.join(os.path.dirname(__file__), 'VERSION')) as vfile:  # pragma nocover
    __version__ = vfile.read().strip()
```
should resolve it.

**Comment by CleanCut** [comment]
Good catch, thanks!

**Comment by CleanCut** [comment]
Fix is in 2.13.1 (just released).
